src/neuroglancer/datasource/precomputed/frontend.ts

```diff
@@ -16,16 +16,19 @@
 
 import {ChunkManager} from 'neuroglancer/chunk_manager/frontend';
 import {registerDataSourceFactory} from 'neuroglancer/datasource/factory';
-import {MeshSourceParameters, VolumeChunkEncoding, VolumeChunkSourceParameters} from 'neuroglancer/datasource/precomputed/base';
+import {MeshSourceParameters, SkeletonSourceParameters, VolumeChunkEncoding, VolumeChunkSourceParameters} from 'neuroglancer/datasource/precomputed/base';
+import {parameterizedSkeletonSource} from 'neuroglancer/skeleton/frontend';
```

Comment:
Can you rename this to `defineParameterizedSkeletonSource` - then it matches the other sources, and it's less likely to be confused with `ParamerterizedSkeletonSource` in `neuroglancer/skeleton/backend`

Reply:
good idea